### PR TITLE
Adding functional tests

### PR DESCRIPTION
**Pull request by fao89**

### CODE REVIEW

**Review by rochacbruno — APPROVED** [review]

File: galaxy_ng/tests/functional/cli/test_collection_upload.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+    def test_upload_collection(self):
+        """Test whether ansible-galaxy can upload a Collection to Pulp."""
+        delete_orphans()
+
+        data = str(self.namespace_api.list().data)
+        if "pulp" not in data:
+            self.namespace_api.create(namespace={"name": "pulp", "groups": []})
+
+        temp_path = f"/tmp/{uuid4()}"
```
Comment (awcrosby) [inline galaxy_ng/tests/functional/cli/test_collection_upload.py:56]: could use a temp dir in a context manager to cleanup file afterwards `with tempfile.TemporaryDirectory() as tmp_dir:`

**Review by awcrosby — APPROVED** [review]
minor comment, LGTM

**Comment by fao89** [comment]
👀

**Comment by fao89** [comment]
Can I have this merged?

**Pull request merged**